src/components/panels/details/info-multiple-selection.tsx

```diff
@@ -1,46 +1,127 @@
 import { Box, Button, Heading, VStack } from '@chakra-ui/react';
 import React from 'react';
-import { useTranslation } from 'react-i18next';
+import { Theme } from '../../../constants/constants';
+import { LineStyleType } from '../../../constants/lines';
 import { StationAttributes } from '../../../constants/stations';
+import { useTranslation } from 'react-i18next';
+import { saveGraph } from '../../../redux/param/param-slice';
 import { useRootDispatch, useRootSelector } from '../../../redux';
-import { addSelected, clearSelected } from '../../../redux/runtime/runtime-slice';
+import {
+    openPaletteAppClip,
+    setRefreshEdges,
+    setRefreshNodes,
+    setSelected,
+} from '../../../redux/runtime/runtime-slice';
+import { AttributesWithColor } from './color-field';
+import { RmgFields } from '@railmapgen/rmg-components';
+import ThemeButton from '../theme-button';
 
 export default function InfoMultipleSection() {
     const { t } = useTranslation();
     const dispatch = useRootDispatch();
-    const { selected } = useRootSelector(state => state.runtime);
+    const {
+        selected,
+        theme,
+        paletteAppClip: { output },
+    } = useRootSelector(state => state.runtime);
+
+    const hardRefresh = React.useCallback(() => {
+        dispatch(setRefreshNodes());
+        dispatch(setRefreshEdges());
+        dispatch(saveGraph(graph.current.export()));
+    }, [dispatch, setRefreshNodes, setRefreshEdges, saveGraph]);
     const graph = React.useRef(window.graph);
 
-    const handleChange = (node: string) => {
-        dispatch(clearSelected());
-        dispatch(addSelected(node));
+    const handleChange = (id: string) => {
+        dispatch(setSelected([id]));
+    };
+
+    const getName = (id: string) => {
+        if (id.startsWith('stn') || id.startsWith('misc_node')) {
+            const attr = graph.current.getNodeAttributes(id);
+            const type = attr.type;
+            return id.startsWith('stn') ? (attr[type] as StationAttributes).names.join('/') : type;
+        } else if (id.startsWith('line')) {
+            const [s, t] = graph.current.extremities(id);
+            const source = graph.current.getSourceAttributes(id);
+            const target = graph.current.getTargetAttributes(id);
+            const sT = source.type;
+            const tT = target.type;
+            return (
+                (s.startsWith('stn') ? (source[sT] as StationAttributes).names[0] : sT) +
+                ' - ' +
+                (t.startsWith('stn') ? (target[tT] as StationAttributes).names[0] : tT)
+            );
+        }
+    };
+
+    const handleChangeColor = (color: Theme) => {
+        selected.forEach(id => {
+            if (graph.current.hasEdge(id)) {
+                const thisType = graph.current.getEdgeAttributes(id).style;
+                const attrs = graph.current.getEdgeAttribute.bind(graph.current)(id, thisType);
```

Comment:
No more bind here. `const attrs = graph.current.getEdgeAttribute(id, thisType);` should work :)